Makefile.include

```diff
@@ -121,6 +124,9 @@ override DLCACHE_DIR            := $(abspath $(DLCACHE_DIR))
 EXTERNAL_BOARD_DIRS             := $(foreach dir,\
                                              $(EXTERNAL_BOARD_DIRS),\
                                              $(abspath $(dir)))
+EXTERNAL_MODULE_DIRS            := $(foreach dir,\
+                                             $(EXTERNAL_MODULE_DIRS),\
+                                             $(abspath $(dir)))
```

Comment:
Here any entries in `EXTERNAL_MODULE_DIRS` that are relative will be converted to absolute paths.